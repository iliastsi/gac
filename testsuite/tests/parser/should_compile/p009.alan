utf8() : proc
{
(* θέλουμε να ελέγξουμε την υποστήριξη utf8 encoded string *)
  writeString("Έχουμε υποστήριξη UTF-8\n");
  writeString("Γιούπι!!!");
}
