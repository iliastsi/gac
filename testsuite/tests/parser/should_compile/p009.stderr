==================== Parser ====================
utf8() : proc
{ -- utf8
  writeString("Έχουμε υποστήριξη UTF-8\n");
  writeString("Γιούπι!!!");
} -- utf8


